services/core-api/app/api/mines/project_summary/resources/project_summary_list.py

```diff
@@ -0,0 +1,66 @@
+from flask_restplus import Resource, inputs
+from werkzeug.exceptions import NotFound
+from decimal import Decimal
+
+from app.extensions import api
+from app.api.utils.resources_mixins import UserMixin
+from app.api.utils.custom_reqparser import CustomReqparser
+from app.api.utils.access_decorators import requires_any_of, VIEW_ALL, MINESPACE_PROPONENT
+from app.api.mines.mine.models.mine import Mine
+from app.api.mines.project_summary.response_models import PROJECT_SUMMARY_MODEL
+from app.api.mines.project_summary.models.project_summary import ProjectSummary
+
+
+class ProjectSummaryListResource(Resource, UserMixin):
```

Comment:
jus a general comment but this resource class and the ProjectSummary resource class do not need to be in separate files. you can have a project_summary_resource that contains the List and normal resources.